radix-engine/src/transaction/transaction_receipt.rs

```diff
@@ -727,7 +727,12 @@ impl<'a> ContextualDisplay<TransactionReceiptDisplayContext<'a>> for Transaction
                     vault_id.display(address_display_context),
                     resource.display(address_display_context),
                     match delta {
-                        BalanceChange::Fungible(d) => format!("{}", d),
+                        BalanceChange::Fungible {
```

Comment:
I'd potentially rather we use our own ResolvedBalanceChange model in the node, and potentially make this optional or remove it, although might be worth chatting to Yulong.

Reply:
Line 725 makes it clear that we render a balance change (and a resulting balance now) for a specific Vault.

(because Engine cannot calculate account's [or any global entity's] balance change)